playbooks/ldap-validator/viyaldapvalidator.yml

```diff
@@ -43,8 +43,8 @@
   - name: Assert that the most important variables are defined
     assert:
       that:
-        - "{{ item }} is defined"
-        - "{{ item }} | trim != '' "
+        - "item is defined"
+        - "item | trim != '' "
       msg: |
         This item is not defined or empty in your sitedefault file: 
         {{item}}  
```

Comment:
Would this reference need to be updated too?

Reply:
I think it is only conditionals that cannot use jinja2 template delimeters

Reply:
I assume you mean the {{item}} in the msg:     From what I've seen with other instances of the issue, it's not every usage of the delimiters.   It seems to be only when directly used in assert: logic.